src/test/testMergeSchemas.ts

```diff
@@ -88,35 +137,76 @@ const linkSchema = `
 
 
   extend type Booking implements Node {
-    # The property of the booking.
+    """
+    The property of the booking.
+    """
     property: Property
   }
 
   extend type Property implements Node {
-    # A list of bookings.
+    """
+    A list of bookings.
+    """
     bookings(
-      # The maximum number of bookings to retrieve.
+      """
+      The maximum number of bookings to retrieve.
+      """
       limit: Int
     ): [Booking]
   }
 
   extend type Query {
     delegateInterfaceTest: TestInterface
     delegateArgumentTest(arbitraryArg: Int): Property
-    # A new field on the root query.
+    """
+    A new field on the root query.
+    """
     linkTest: LinkType
     node(id: ID!): Node
     nodes: [Node]
   }
 
   extend type Customer implements Node ${graphql11compat}
 `;
+if (process.env.GRAPHQL_VERSION === '^0.11') {
+  graphql11compat = '{}';
```

Comment:
This doesn't need to be a separate variable anymore, right? could just be in the string below